oci.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import logging
+
+from oci import config
+from oci import gerrit
+from oci import jenkins
```

Comment:
I think we should move this to oci/main.py.

The way to run the tool is to have a script called "oci", that will do:

```
#!/usr/bin/python
from oci import main
main.run()
```

This file will be installed in the path to make the tool available from the shell, while the oci
package is installed in python site-packages directory.

Reply:
Done, but please see my comment on this.